docker-compose.yml

```diff
@@ -3,8 +3,12 @@ version: "3.2"
 services:
 
   curation.app:
-    container_name: agr.curation.api.server
+    container_name: agr.curation.${NET}.api.server
     image: 100225593120.dkr.ecr.us-east-1.amazonaws.com/agr_curation:${AGR_CURATION_RELEASE}
+    logging:
+      driver: gelf
+      options:
+        gelf-address: 'udp://logs.alliancegenome.internal:12201'
```

Comment:
PS Plus I have tried these gelf lines in the past and for some reason the docker on the EBS instance does not pass the log data on to the log server.

Reply:
This worked fine for me (see logtrail and search for "quarkus"), however the container name does not show up in the system list for some reason (something we've noticed before with some other containers).